lib/Init.vala

```diff
@@ -61,7 +61,7 @@ namespace Granite {
 
         Gtk.StyleContext.remove_provider_for_display (display, accent_provider);
         accent_provider.load_from_string ("@define-color accent_color %s;".printf (accent_color));
-        Gtk.StyleContext.add_provider_for_display (display, accent_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+        Gtk.StyleContext.add_provider_for_display (display, accent_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION - 1);
```

Comment:
I'd use THEME + 1 instead

Reply:
I tried THEME + 1 but it didn't work for some reason

Reply:
Huh. THat is strange.